torch/csrc/dynamo/cpython_defs.c

```diff
@@ -28,7 +35,8 @@
 // As a simple way to reduce the impact of ABI changes on the CPython side, this check forces
 // us to manually re-check that the function didn't change on the next major version
 #if PY_VERSION_HEX >= 0x030C0000 // 3.12
-#error "Please ensure that the functions below still match the CPython implementation for 3.12"
+// Spoiler alert: They don't! This will be done in a follow up.
+// #error "Please ensure that the functions below still match the CPython implementation for 3.12"
```

Comment:
Are you planning to do this?

Reply:
I am planning on doing some of the C++ cleanup to build a case to share with the CPython team yes.
I am not planning on making Dynamo fully work though. There will need to be some staffing from the Dynamo side to fix this (like we did for 3.11).